src/components/Header.jsx

```diff
@@ -130,10 +142,36 @@ function Header() {
           <HeaderNavMenu links={links} />
         </Grid>
 
+        <Grid item>
+          <IconButton onClick={(e) => setAnchorEl(e.currentTarget)}>
+            <LanguageIcon />
+          </IconButton>
+        </Grid>
+
         <Grid item className={classes.authItem}>
-          <AuthZoomButton />
+          <AuthZoomButton t={t} />
         </Grid>
       </Grid>
+
+      <Menu
+        open={Boolean(anchorEl)}
+        anchorEl={anchorEl}
+        onClose={() => setAnchorEl(null)}
+        anchorOrigin={{ horizontal: 'center', vertical: 'bottom' }}
+        transformOrigin={{ horizontal: 'center', vertical: 'top' }}>
+        {Object.entries(languages).map(([k, v], index) => (
+          <MenuItem
+            key={k}
+            selected={index === selectedIndex}
```

Comment:
instead of using `index` and `selectedIndex` can you compare `v` with the current language?

Reply:
We could do this in a future PR where we support the language via params since I don't think there is a method that outputs what language is being used.

Reply:
Nice